client/src/components/RatingsAndReviews/Reviews/Helpful.jsx

```diff
@@ -30,7 +31,10 @@ export default function helpful({review}) {
 
   return (
     <SmallFont>
-      <span>Helpful? <Button onClick={handleClick}>Yes ({helpfulCount})</Button> <Button onClick={handleReport}>Report</Button></span>
+      <span>Helpful?
+        <Button onClick={handleClick}>Yes ({helpfulCount})</Button>
```

Comment:
change name to something more descriptive like 'handleClickYes', there's a lot of handleClick functions throughout the app